pkg/resources/manifests/deploy.yaml

```diff
@@ -117,6 +117,12 @@ spec:
             exec:
               command:
                 - "/cleanup.sh"
+        readinessProbe:
+          periodSeconds: 5
+          exec:
+            command:
+              - /bin/gadgettracermanager
+              - -liveness
```

Comment:
nit: Should we change the flag name from `liveness` to `health`?

Reply:
I think we can skip this change for now.

Reply:
Yeah, sure.